docs/configuration/README.md

```diff
@@ -663,23 +663,11 @@ The `limits_config` block configures global and per-tenant limits for ingesting
 logs in Loki.
 
 ```yaml
-# Per-user ingestion rate limit in samples per second.
-[ingestion_rate: <float> | default = 25000]
+# Per-user ingestion rate limit in samples per second. Units in MB.
+[ingestion_rate_mb: <float> | default = 4]
 
-# Per-user allowed ingestion burst size (in number of samples).
-[ingestion_burst_size: <int> | default = 50000]
-
-# Whether or not, for all users, samples with external labels
-# identifying replicas in an HA Prometheus setup will be handled.
-[accept_ha_samples: <boolean> | default = false]
-
-# Prometheus label to look for in samples to identify a
-# Prometheus HA cluster.
-[ha_cluster_label: <string> | default = "cluster"]
-
-# Prometheus label to look for in samples to identify a Prometheus HA
-# replica.
-[ha_replica_label: <string> | default = "__replica__"]
+# Per-user allowed ingestion burst size (in number of samples). Units in MB.
```

Comment:
Same
```suggestion
# Per-user allowed ingestion burst size (in sample size). Units in MB. Warning, very high limits will be reset every -distributor.limiter-reload-period.
```

Reply:
I changed this just a little bit: I changed it from using the flag name to the YAML name.